pandas/tools/util.py

```diff
@@ -173,6 +174,8 @@ def to_numeric(arg, errors='raise', downcast=None):
             values = arg.values
     elif isinstance(arg, (list, tuple)):
         values = np.array(arg, dtype='O')
+    elif isinstance(arg, Decimal):
```

Comment:
If I update the isscalar function we get different functionality depending on if you do
```python
df['a'] = df['a'].apply(is_numeric)
df['b'] = df['b'].apply(is_numeric)
```
vs

```python
df[['a', 'b']] = df[['a', 'b']].apply(is_numeric)
```
In the first case a column with a Decimal in it will remain as type object it's decimal objects will remain as decimal objects

In the second case columns with Decimal will be converted to float64